### PR TITLE
Merge paradox/scala/cluster-sharding.md and java/cluster-sharding.md 

### PR DESCRIPTION
**Pull request by richardimaoka**
Refs #23052

## Rendered HTML:

- https://richard-akka-doc.firebaseapp.com/scala/cluster-sharding.html
- https://richard-akka-doc.firebaseapp.com/java/cluster-sharding.html

## I split this in two commits:

- **1st commit:** Identical contents in scala/cluster-sharding.md and java/cluster-sharding.md -> **this is to take the diff both in scala/cluster-sharding.md and java/cluster-sharding.md**

  - [scala/cluster-sharding.md  (prev) vs. scala/cluster-sharding.md (after)](https://github.com/richard-imaoka/akka/commit/317ea1bbb43be29d8912234aca0608b71f2055ee#diff-ed8eed6757cc63ff7243bcad0c3b47df2)
  - [java/cluster-sharding.md (prev) vs. scala/cluster-sharding.md (after)](https://github.com/richard-imaoka/akka/commit/317ea1bbb43be29d8912234aca0608b71f2055ee#diff-4db94d967db69823cdc47a430618e1d2)

- **2nd commit:** Replace java/cluster-sharding.md with a symlink to scala/cluster-sharding.md

### CODE REVIEW

File: akka-docs/src/main/paradox/java/cluster-sharding.md [hunk 1]
```diff
@@ -48,37 +52,45 @@ When using the sharding extension you are first, typically at system startup on
...
 
 This example illustrates two different ways to define the entity identifier in the messages:
 
  * The `Get` message includes the identifier itself.
  * The `EntityEnvelope` holds the identifier, and the actual message that is
 sent to the entity actor is wrapped in the envelope.
 
-Note how these two messages types are handled in the `entityId` and `entityMessage` methods shown above.
-The message sent to the entity actor is what `entityMessage` returns and that makes it possible to unwrap envelopes
+Note how these two messages types are handled in the Sscala[`extractEntityId` function] @java[`entityId` and `entityMessage` methods] shown above.
```
Comment (raboof) [inline akka-docs/src/main/paradox/java/cluster-sharding.md]: typo: `@scala`

File: akka-docs/src/main/paradox/java/cluster-sharding.md [hunk 2]
```diff
@@ -48,37 +52,45 @@ When using the sharding extension you are first, typically at system startup on
...
+The message sent to the entity actor is @scala[the second part of the tuple return by the `extractEntityId`] @[what `entityMessage` returns] and that makes it possible to unwrap envelopes
```
Comment (raboof) [inline akka-docs/src/main/paradox/java/cluster-sharding.md]: 'returned' perhaps?

**Review by raboof — CHANGES_REQUESTED** [review]
Nice, one super tiny typo and then this is good to go!

**Comment by richard-imaoka** [comment]
Thanks @raboof for quick review. Fixed the text in the new commit.

**Pull request merged**